### ISSUE TITLE
Add jabber/xmpp feature

### ISSUE THREAD

**Issue by dveeden** [etsy/morgue#100]
The IRC support is nice, but jabber/xmpp would be awesome :smiley:

**Comment by alanbuxey** [comment]
+1

would also like Slack  :)

**Comment by chris-lmn** [comment]
Yes please, would use this if it worked with Slack! :) please and thank you.
